plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -120,10 +120,37 @@ protected function render( $attributes, $content, $block ) {
 			)
 		);
 
-		$number_of_items_in_cart  = $this->get_cart_item_quantities_by_product_id( $product->get_id() );
+		$is_grouped              = $product->is_type( 'grouped' );
+		$grouped_product_ids     = $product->get_children();
+		$is_product_purchasable  = $product->is_purchasable();
+		$number_of_items_in_cart = $this->get_cart_item_quantities_by_product_id( $product->get_id() );
+
+		if ( $is_grouped ) {
+			$children = $product->get_children();
```

Comment:
Nit: `$grouped_product_ids` and `$children` hold the same value, no? I guess we can remove one of the two variables.